### PR TITLE
[CLDC-1467] add type of building question to sales

### PR DESCRIPTION
**Pull request by Ted-U**
https://digital.dclg.gov.uk/jira/browse/CLDC-1467

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -338,7 +339,7 @@
     t.integer "age1_known"
     t.string "sex1"
     t.integer "buy1livein"
-    t.integer "buylivein"
```
Comment (baarkerlounger) [inline db/schema.rb]: Why is this field being removed? I don't think this is correct, you might need to rebase and migrate.

Comment (Ted-U) [reply db/schema.rb]: Thanks Dan, fixed now.

**Comment by baarkerlounger** [comment]
@Ted-U is this a duplicate of https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/918/files now and if so can it be closed?

**Comment by ghost** [comment]
@baarkerlounger picking up on the question you asked last to Ted. It seems like it was a duplicate of https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/918 but the page ID was not correct so I fixed it in this PR

**Review by jamesrose — APPROVED** [review]

**Pull request merged**